ogr/services/gitlab/service.py

```diff
@@ -128,3 +128,53 @@ def project_create(
         return GitlabProject(
             repo=repo, namespace=namespace, service=self, gitlab_repo=new_project
         )
+
+    def list_projects(
+        self,
+        namespace: str = None,
+        user: str = None,
+        search_pattern: str = None,
+        language: str = None,
+    ) -> List[GitProject]:
+
+        if namespace:
+            group = self.gitlab_instance.groups.get(namespace)
+            projects = group.projects.list(all=True)
+        if user:
```

Comment:
I would probably use `elif` here and in case of setting both `namespace` and `user` raise an exception, we should have `OperationNotSupported`

Reply:
Thanks for clearing this. I actually thought about how to handle this but couldn't come up with a way. I will update the code accordingly, thanks!